crates/bevy_render/src/camera/camera.rs

```diff
@@ -8,25 +12,90 @@ use bevy_ecs::{
     reflect::ReflectComponent,
     system::{QuerySet, Res},
 };
-use bevy_math::{Mat4, Vec2, Vec3};
+use bevy_math::{Mat4, UVec2, Vec2, Vec3};
 use bevy_reflect::{Reflect, ReflectDeserialize};
 use bevy_transform::components::GlobalTransform;
+use bevy_utils::HashSet;
 use bevy_window::{WindowCreated, WindowId, WindowResized, Windows};
 use serde::{Deserialize, Serialize};
+use wgpu::Extent3d;
 
 #[derive(Component, Default, Debug, Reflect)]
 #[reflect(Component)]
 pub struct Camera {
     pub projection_matrix: Mat4,
     pub name: Option<String>,
     #[reflect(ignore)]
-    pub window: WindowId,
+    pub target: RenderTarget,
     #[reflect(ignore)]
     pub depth_calculation: DepthCalculation,
     pub near: f32,
     pub far: f32,
 }
 
+#[derive(Debug, Clone, Reflect, PartialEq, Eq, Hash)]
+pub enum RenderTarget {
+    /// Window to which the camera's view is rendered.
+    Window(WindowId),
+    /// Image to which the camera's view is rendered.
+    Image(Handle<Image>),
+}
+
+impl Default for RenderTarget {
+    fn default() -> Self {
+        Self::Window(Default::default())
+    }
+}
+
+impl RenderTarget {
+    pub fn get_texture_view<'a>(
+        &self,
+        windows: &'a ExtractedWindows,
+        images: &'a RenderAssets<Image>,
+    ) -> Option<&'a TextureView> {
+        match self {
+            RenderTarget::Window(window_id) => windows
+                .get(window_id)
+                .and_then(|window| window.swap_chain_texture.as_ref()),
+            RenderTarget::Image(image_handle) => {
+                images.get(image_handle).map(|image| &image.texture_view)
+            }
+        }
+    }
+    pub fn get_physical_size(&self, windows: &Windows, images: &Assets<Image>) -> Option<UVec2> {
+        match self {
+            RenderTarget::Window(window_id) => windows
+                .get(*window_id)
+                .map(|window| UVec2::new(window.physical_width(), window.physical_height())),
+            RenderTarget::Image(image_handle) => images.get(image_handle).map(|image| {
+                let Extent3d { width, height, .. } = image.texture_descriptor.size;
+                UVec2::new(width, height)
+            }),
+        }
+    }
+    pub fn get_logical_size(&self, windows: &Windows, images: &Assets<Image>) -> Option<Vec2> {
+        match self {
+            RenderTarget::Window(window_id) => windows
+                .get(*window_id)
+                .map(|window| Vec2::new(window.width(), window.height())),
+            RenderTarget::Image(image_handle) => images.get(image_handle).map(|image| {
+                let Extent3d { width, height, .. } = image.texture_descriptor.size;
+                Vec2::new(width as f32, height as f32)
+            }),
+        }
+    }
+    pub fn is_modified(
```

Comment:
If this follows standard Changed semantics, I think this should probably be called `is_changed`. If it doesn't, it should have some brief docs explaining how it's different.

Reply:
I've added documentation.